src/main.ts

```diff
@@ -49,24 +50,50 @@ async function run() {
       throw error;
     }
 
-    const event = status > 0 ? "REQUEST_CHANGES" : "COMMENT";
+    const failed = status > 0;// ? "REQUEST_CHANGES" : "COMMENT";
+    if (failed) {
+      setFailed("Failed");
+    }
     const body = [
-      `## [size-limit](${SIZE_LIMIT_URL}) report`,
+      SIZE_LIMIT_HEADING,
       table(limit.formatResults(base, current))
     ].join("\r\n");
 
-    try {
-      octokit.pulls.createReview({
-        ...repo,
-        // eslint-disable-next-line camelcase
-        pull_number: pr.number,
-        event,
-        body
-      });
-    } catch (error) {
-      console.log(
-        "Error creating PR review. This can happen for PR's originating from a fork without write permissions."
-      );
+    const { data: commentList } = await octokit.issues.listComments({
```

Comment:
As @dcwither mentioned, maybe using `pagination` here would be better (see [docs](https://octokit.github.io/rest.js/v18#pagination))